server/ws/plugin_adapter.go

```diff
@@ -476,7 +480,7 @@ func (pa *PluginAdapter) BroadcastCategoryChange(category model.Category) {
 		Category: &category,
 	}
 
-	pa.sendTeamMessage(websocketActionUpdateCategory, category.TeamID, utils.StructToMap(message))
+	pa.sendUserMessageSkipCluster(websocketActionUpdateCategory, utils.StructToMap(message), category.UserID)
```

Comment:
Why are we skipping the cluster here? I don't think we should skip the cluster propagation here.

Reply:
Added now.